.circleci/config.yml

```diff
@@ -68,22 +71,11 @@ jobs:
           command: |
             cd examples/vue-kitchen-sink
             yarn build-storybook
-
       - run:
-          name: "Run react kitchen-sink"
+          name: "Run image snapshots on react kitchen-sink"
           command: |
             cd examples/cra-kitchen-sink
-            yarn storybook
```

Comment:
Let's run `yarn storybook --smoke-test` (without `background: true`) just to check that devserver starts successfully)

Reply:
Allright, will re-add this.